plugins/woocommerce-admin/client/layout/store-alerts/index.js

```diff
@@ -264,6 +266,15 @@ export class StoreAlerts extends Component {
 							</Button>
 						</div>
 					) }
+					<Button
+						onClick={ async () => {
+							await removeNote( alert.id );
+							this.previousAlert();
+							invalidateResolutionForStoreSelector( 'getNotes' );
```

Comment:
It would be nice to move this to a function outside of the JSX, to keep the code a bit more readable.